conformance/src/bank_methods.zig

```diff
@@ -0,0 +1,273 @@
+const sig = @import("sig");
+const std = @import("std");
+
+const builtins = @import("builtins.zig");
+
+const Allocator = std.mem.Allocator;
+
+const bincode = sig.bincode;
+const features = sig.core.features;
+const program = sig.runtime.program;
+
+const AccountsDb = sig.accounts_db.AccountsDB;
+
+const Account = sig.core.Account;
+const Pubkey = sig.core.Pubkey;
+const Slot = sig.core.Slot;
+
+const AccountSharedData = sig.runtime.AccountSharedData;
+const FeatureSet = sig.core.features.FeatureSet;
+
+const failing_allocator = sig.utils.allocators.failing.allocator(.{});
+
+/// A minimal implementation of `Bank::apply_feature_activations` for fuzzing purposes.
+/// If a fixture hits an error, we may need to implement the missing feature activation logic.
+/// https://github.com/firedancer-io/agave/blob/10fe1eb29aac9c236fd72d08ae60a3ef61ee8353/runtime/src/bank.rs#L6453
+pub fn applyFeatureActivations(
+    allocator: Allocator,
+    slot: u64,
+    feature_set: *FeatureSet,
+    accounts_db: *AccountsDb,
+    allow_new_activations: bool,
+) !void {
+    const new_feature_activations = try computeActiveFeatureSet(
+        allocator,
+        slot,
+        feature_set,
+        accounts_db,
+        allow_new_activations,
+    );
+
+    for (new_feature_activations.keys()) |feature_id| {
+        const db_account = try tryGetAccount(accounts_db, feature_id) orelse continue;
+        defer db_account.deinit(allocator);
+
+        const activation_slot = try featureActivationSlotFromAccount(
+            allocator,
+            db_account,
+        ) orelse continue;
+
+        const account = try accountSharedDataFromAccount(allocator, &db_account);
+        defer account.deinit(allocator);
+
+        _ = try bincode.writeToSlice(account.data, activation_slot, .{});
+
+        try accounts_db.putAccount(slot, feature_id, account);
+    }
```

Comment:
This appears to have no effect.

1. `new_feature_activations` are the features that were found to have no activation slot, so each loop iteration will continue before writing anything to accountsdb.
2. Even if it *did* find an activation slot, all this does is re-write the same value it just read from the account, which is basically still a noop.